src/cocoa_joystick.m

```diff
@@ -469,13 +481,13 @@ void _glfwTerminateJoysticksNS(void)
 int _glfwPlatformJoystickPresent(int joy)
 {
     _GLFWjoydeviceNS* joystick = _glfw.ns_js.devices + joy;
-    return pollJoystickEvents(joystick);
+    return joystick->present ? GLFW_TRUE : GLFW_FALSE;
```

Comment:
I assumed that was the case, but since the code in cocoa_joystick.m typically evaluated joystick->present as a compiler-time boolean (e.g., if (joystick->present)) I thought it was safer to be explicit since a user might evaluate the return value specifically with GLFW_TRUE.

Coming from a C++ background, I exclusively use enumerated values rather than #defines or constants since the compiler will automatically restrict variables to their enumerated values (unless you explicitly typecast, after which I always include an assert statement to confirm that the variable is still within the limits of the enumeration); also, there are additional benefits such as the compiler warning you if any enumerated values are missing within a switch statement.  However, when I program in C, I try to be more careful and explicit.